go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/spacelift-io/spacectl
 
-go 1.22
+go 1.23.4
```

Comment:
`github.com/mark3labs/mcp-go` requires go 1.23. Are we ok with this change?

Reply:
Sure.

Reply:
Should we bump the major version on release?

Reply:
It is already bumped on main